src/thirdparty/CMakeLists.txt

```diff
@@ -19,11 +19,11 @@ target_include_directories(cli11 SYSTEM INTERFACE
             $<BUILD_INTERFACE:${PROJECT_SOURCE_DIR}/thirdparty>)
 
 install(TARGETS              cli11
-        EXPORT               cli11-targets
+        EXPORT               axom-targets
         INCLUDES DESTINATION include)
 install(DIRECTORY   ${PROJECT_SOURCE_DIR}/thirdparty/CLI11
         DESTINATION include )
-install(EXPORT cli11-targets DESTINATION lib/cmake)
+install(EXPORT axom-targets DESTINATION lib/cmake)
```

Comment:
@white238 -- I changed this so that all exports are placed within a single ``axom-targets`` file.